gf-webapp/src/logic/rooms/roomsPage/roomBlock/roomItem/information/Information.tsx

```diff
@@ -23,7 +26,8 @@ export const Information = (props: InformationProps) => {
         {dictionary.roomInfo.adults}
       </div>
       <div className={styles.description}>
-        TODO: add room dynamic description from dictionary
+        {dictionary.roomInfo.roomDescription[props.roomTitle]}
+        {/* TODO: add room dynamic description from dictionary */}
```

Comment:
Delete comments  if it's works. 
Or it doesn't work?

Reply:
Done